source/pacmap/pacmap.py

```diff
@@ -869,9 +869,11 @@ def __init__(self,
             self.random_state = 0
             _RANDOM_STATE = None  # Reset random state
 
-        if self.n_components < 2:
+        if self.n_components < 1:
             raise ValueError(
                 "The number of projection dimensions must be at least 2.")
```

Comment:
It may be better to edit the error message to "at least 1" to be consistent with line 872.